src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -137,8 +137,8 @@ public function exists($files)
      * Sets access and modification time of file.
      *
      * @param string|iterable $files A filename, an array of files, or a \Traversable instance to create
-     * @param int             $time  The touch time as a Unix timestamp
-     * @param int             $atime The access time as a Unix timestamp
+     * @param int|null        $time  The touch time as a Unix timestamp
```

Comment:
We should then add a short description about the behaviour when `null` is passed.

Reply:
Added a short description for $atime too